### PR TITLE
Core 7731Fix UI security bug where sensitive passwords and port numbers are exposed.

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java [hunk 1]
```diff
@@ -211,6 +213,27 @@ public static DEProperties getInstance() {
...
+        keys.add("org.iplantc.discoveryenvironment.muleServiceBaseUrl");
+        keys.add("org.iplantc.services.de-data-mgmt.base");
+        keys.add("org.iplantc.services.permIdRequests");
+        keys.add("org.iplantc.services.permIdRequests");
+        keys.add( "org.iplantc.services.admin.de-data-mgmt.base");
+        keys.add("org.iplantc.services.file-io.base.secured");
+        keys.add("org.iplantc.discoveryenvironment.notifications.poll-interval");
+        keys.add("org.iplantc.discoveryenvironment.contextMenu.enabled");
+        keys.add("org.iplantc.communitydata.path");
+        return keys;
```
Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java]: Are these all the keys that the UI needs?  I thought the list was longer, like one for the Beta App Category id and one for trash I thought should be in here.

**Comment by ash3rz** [comment]
If all the keys the UI needs are in the list, then this looks ready to me 🚢

File: ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java (hunk 1 above)
Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java]: I picked the property list from whats listed in the DEProperties class. Hmm let me check belphegor side once.

Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java]: Seems like thats the list.

Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java]: When I go to Belphegor with these changes, I get an exception that seems to stem from not finding `org.iplantc.discoveryenvironment.keepalive.interval`, so that key at least is missing.

Also, can we switch from adding the strings directly to using the variables they're assigned to, like `COMMUNITY_DATA_PATH` instead of `org.iplantc.communitydata.path`?

Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/shared/DEProperties.java]: I have submitted another PR that relates to deleting all the code that involves KeepAliveTimer. Add variables instead of strings sounds like a good plan
